lib/server.js

```diff
@@ -359,7 +359,7 @@ Server.prototype.handleUpgrade = function (req, socket, upgradeHead) {
       return;
     }
 
-    var head = new Buffer(upgradeHead.length); // eslint-disable-line node/no-deprecated-api
```

Comment:
Couldn't it be replaced by `Buffer.from(upgradeHead)` directly?

Reply:
Didn't properly read what the code was doing. Adjusted.